src/app/organizations/manage/reset-password.component.html

```diff
@@ -0,0 +1,78 @@
+<div class="modal fade" tabindex="-1" role="dialog" aria-modal="true" aria-labelledby="resetPasswordTitle">
+    <div class="modal-dialog" role="document">
+        <form class="modal-content" #form (ngSubmit)="submit()" [appApiAction]="formPromise">
+            <div class="modal-header">
+                <h2 class="modal-title" id="resetPasswordTitle">
+                    {{'resetPassword' | i18n}}
+                    <small class="text-muted" *ngIf="name">{{name}}</small>
+                </h2>
+                <button type="button" class="close" data-dismiss="modal" appA11yTitle="{{'close' | i18n}}">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <app-callout type="warning">{{'resetPasswordLoggedOutWarning' | i18n: loggedOutWarningName}}
+                </app-callout>
+                <app-callout type="info" *ngIf="enforcedPolicyOptions">
```

Comment:
It feels like we are duplicating this callout in many places, could we refactor it to be a component with `enforcedPolicyOptions` as input?